File: Python/Sorting/SelectionSorting.py
arr=list(map(int,input().split()))
for i in range(0,len(arr)-1):
    min=i
    for j in range(i+1,len(arr)):
        if(arr[min]>arr[j]):
            min=j
    if(min!=i):
        arr[i],arr[min]=arr[min],arr[i]
print(arr)

File: JAVA/Algorithms/Sorting/SelectionSorting.java
package Algorithms.Sorting;

public class SelectionSorting {
    public static Integer[] sort(Integer[] arr)
    {
        int n=arr.length;
        for(int i=0;i<n-1;i++)
        {
            int min=i;
            for(int j=i+1;j<n;j++)
            {
                if(arr[min]>arr[j])
                    min=j;
            }
            if(min!=i)
            {
                int temp=arr[i];
                arr[i]=arr[min];
                arr[min]=temp;
            }
        }
        return arr;
    }
}
